x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/overrides.js

```diff
@@ -55,6 +59,11 @@ export class Overrides extends Component {
     }
   });
 
+  customTimestampFormatErrors = i18n.translate('xpack.ml.fileDatavisualizer.editFlyout.overrides.customTimestampFormatErrorMessage', {
+    defaultMessage: `Timestamp format must be a combination of these Java date/time formats:
+      yy, yyyy, M, MM, MMM, MMMM, d, dd, EEE, EEEE, H, HH, h, mm, ss, a, XX, XXX, zzz`
```

Comment:
It would be good to mention that `S` through `SSSSSSSSS` is also supported, but only after `ss`.  It's tricky to think of a nice way to phrase this without being overly verbose, but I think if we don't mention it at all people will get frustrated as it will look like fractional seconds are not supported.

Reply:
I've added `S through SSSSSSSSS` to the message here but since we also have the specific error for the `only after ss` part - it should be enough information.

Reply:
Added in 2adb1002d6ef80fb467803db00d1273545636951